test/src/main/python/connections.py

```diff
@@ -0,0 +1,14 @@
+from locust import HttpUser, between, task
+
+# Locust Laodtests
+#
+# Install: pip3 install locust
+# Run: locust -f connections.py
+#
+# See: https://docs.locust.io/en/stable/index.html
+class WebsiteUser(HttpUser):
+    # wait_time = between(5, 15)
+
+    @task
+    def fruit(self):
+        self.client.get("/shop/products/")
```

Comment:
Was this file added by accident?

Reply:
No. But it is somewhat hidden. This is a locust script to provoke the limit from above. And it is useful for loadtesting. I'll move it and add some documentation.